lease/util.go

```diff
@@ -0,0 +1,37 @@
+// Copyright 2018 The etcd Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package lease
+
+import (
+	"encoding/binary"
+	"fmt"
+)
+
+const idBytesLen = 8
+
+// IDToBytes encodes int64 lease ID in big-endian binary format.
+func IDToBytes(n int64) []byte {
```

Comment:
Similar comments as above... Hiding allocations/conversions in this manner can cause performance issues if this code is called frequently.